go.mod

```diff
@@ -1,12 +1,15 @@
 module github.com/ForceCLI/force
 
-go 1.20
+go 1.22
+
+toolchain go1.22.4
```

Comment:
The docker image in .circleci/config.yml will need to be updated to support this.